deposit/migrations/0013_populate_licenses.py

```diff
@@ -0,0 +1,105 @@
+from django.db import migrations
+
```

Comment:
Don't you think it would be worth splitting this into one migration per protocol? I would add them to `deposit/zenodo/migrations` and `deposit/osf/migrations`, so that the migrations are only run if these apps are enabled in the settings. That would also help keeping repository-specific things in their own subfolders, and demonstrate to a third-party how to add support for another protocol without interferring with the generic deposit app.

Reply:
I'll do as follows:
* Non-CC go into zenodo/osf
* LicenseChoosers go into zenodo/osf